pdesolvers/tests/test_black_scholes.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import numpy as np
+
+import pdesolvers.pdes.black_scholes as bse
+import pdesolvers.solvers.black_scholes_solvers as solver
+
+
+class TestBlackScholesEquation:
+    def test_get_option_type(self):
```

Comment:
If you're planning to implement this test later, it's better if you make it fail (remove the `pass` and write something like `assert False`)